.github/workflows/vdscanner_tools_delivery.yml

```diff
@@ -0,0 +1,124 @@
+name: VD Scanner tools Delivery
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on pull request but only changes in src/wazuh_modules/vulnerability_scanner/{tools,src,include,schemas} directories.
+  # Trigger on pull request changes
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
+    paths:
+      - ".github/workflows/vdscanner_tools_delivery.yml"
+
+  # Trigger when a pull request is merged into master
+  push:
+    branches:
+      - master
+    paths:
+      - ".github/workflows/vdscanner_tools_delivery.yml"
+
+  # Allows manual trigger.
+  workflow_dispatch:
+
+# Ensures only one instance of this workflow is running per PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+env:
+  TOOLS_DIR: ${{github.workspace}}/src/wazuh_modules/vulnerability_scanner/build/testtool/
+  TOOLS_FILE_NAME: legacy_vdscanner_testtools
+
+jobs:
+  build:
+    name: VD Scanner Tools Delivery
```

Comment:
I'd consider adding this 
https://github.com/wazuh/wazuh/blob/136ff7abd8da4b6515e9c1a797e3e39bf127a456/.github/workflows/integration-tests-execd-tier-0-1-win.yml#L22

So if update the workflow at `main`, we avoid running a 4.x workflow over the 5.x branch because it'll fail.